mlir/test/Integration/Dialect/SparseTensor/CPU/dense_output.mlir

```diff
@@ -18,18 +18,18 @@
 //--------------------------------------------------------------------------------------------------
 
 // REDEFINE: %{env} = TENSOR0="%mlir_src_dir/test/Integration/data/test.mtx"
```

Comment:
Seems like it would be easier to put the `env` command inside the `%{env}` substitution.

Reply:
I hesitated, but it seemed that this was written so that %{env} can be redefined at will (even though no test is actually doing do at the moment)

Reply:
I don't understand why these tests define and then redefine `%{env}` exactly once.

Either way, it works, so it's up to MLIR people to decide how to organize this.